components/prism-kotlin.js

```diff
@@ -1,15 +1,22 @@
 (function (Prism) {
+
+	var functionName = /(?:`[\w !"#$%^&()*+,=?@{|}~\-]+`|\w+)/.source;
```

Comment:
According to [Kotlin's spec](https://kotlinlang.org/spec/syntax-and-grammar.html#identifiers), it should be this:

```suggestion
	var functionName = /(?:`[^\r\n`]+`|\w+)/.source;
```

With this, the pattern is short enough to be inlined, I think.

Reply:
Fixed